peekingduck/pipeline/nodes/model/posenet.py

```diff
@@ -78,7 +79,9 @@ def run(self, inputs: Dict[str, Any]) -> Dict[str, Any]:
         bboxes, keypoints, keypoint_scores, keypoint_conns = self.model.predict(
             inputs["img"]
         )
-        bbox_labels = ["Person"] * len(bboxes)
+        bbox_labels = np.array(["Person"] * len(bboxes))
```

Comment:
Was this supposed to have been changed to `"person"`?

Reply:
Oops, accidentally undid the change before pushing. Fixed.